roles/ovirt-datacenter-cleanup/tasks/find_hosted_engine_details.yml

```diff
@@ -0,0 +1,31 @@
+- name: Find existing VMs
+  ovirt_vms_facts:
+    auth: "{{ ovirt_auth }}"
+    pattern: "datacenter={{ data_center_name }} and name={{ hosted_engine_vm_name }}"
+    fetch_nested: true
+
+- name: Find HE info if HE
+  set_fact:
+    hosted_engine: "{{ ovirt_vms | selectattr('name', 'equalto', hosted_engine_vm_name) | map(attribute='name') | list != [] }}"
```

Comment:
`selectattr('name', 'equalto', hosted_engine_vm_name)` isn't really needed, it's already done by previous task.
I think all you need is just "{{ ovirt_vms | length > 0 }}"

Reply:
Done